test/ffmpeg-vaapi/decode/10bit/vvc.py

```diff
@@ -0,0 +1,29 @@
+###
+### Copyright (C) 2024 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from .....lib import *
+from .....lib.ffmpeg.vaapi.util import *
+from .....lib.ffmpeg.vaapi.decoder import DecoderTest
+
+spec = load_test_spec("vvc", "decode", "10bit")
+
+@slash.requires(*platform.have_caps("decode", "vvc_10"))
+@slash.requires(*have_ffmpeg_decoder("vvc"))
+class default(DecoderTest):
+  def before(self):
+    # default metric
+    vars(self).update(
+      caps        = platform.get_caps("decode", "vvc_10"),
+      ffdecoder   = "vvc",
+      metric      = dict(type = "ssim", miny = 1.0, minu = 1.0, minv = 1.0),
+    )
```

Comment:
If we use a generic "strict" property in base decoder...
```suggestion
    vars(self).update(
      caps        = platform.get_caps("decode", "vvc_10"),
      ffdecoder   = "vvc",
      metric      = dict(type = "ssim", miny = 1.0, minu = 1.0, minv = 1.0), # default metric
      strict      = -2, # Currently, VVC decoder is experimental. Enable it with strict.
    )
```